### ISSUE TITLE
Authentication error 1001, Invalid Account

### ISSUE THREAD

**Issue by ngdekhai** [tarikhagustia/php-mt5#8]
Hi,

This might be a stupid question but I keep getting error 1001, Invalid Account on authentication.
My questions are
1) the login and password is for manager account's username and password right? Or the password is different type of password, like WebAPI password?

2)the account I received is provided by my client, is there different type of manager account or does he need to enable something for me to access it through api ? Ps. he have changed password twice, yet I cannot access
Thanks for your help.

**Comment by azhar4xportal** [comment]
yes, you need a manager login with an API password. API password is different from normal password. You should read documentation.

**Comment by Javierfernandez93** [comment]
I have the same problem. Hope u can help us

where is the documentation?
